app/controllers/application_controller.rb

```diff
@@ -144,6 +146,17 @@ def initialize_stripe
     Stripe.log_level = Stripe::LEVEL_INFO
   end
 
+  def remember_cookie_sync
+    # Set remember cookie token in case not properly set.
+    if user_signed_in? && cookies[:remember_user_token].blank?
+      current_user.remember_me = true
+      current_user.remember_me!
+      remember_me(current_user)
+    elsif !user_signed_in?
+      cookies.delete :remember_user_token, domain: ".#{SiteConfig.app_domain}"
```

Comment:
Just leaving a note here to ensure this actually works; I recently learned that `cookies.delete`[ doesn't necessarily work the way you'd expect](https://github.com/forem/forem/pull/10211).

Reply:
Yeah, this was just a bunch of trial and error. Not needed in the end.